### PR TITLE
Core Concepts week 1 day 3 update

### PR DESCRIPTION
**Pull request by beth-panx**
---

⚠️ Merge/Release date 2/16

---

Updates:
- Week 1 day 3 article

***Please ignore day 2 changes. They are included in this PR #28**

### CODE REVIEW

File: docs/30DaysOfPWA/core-concepts/03.md [hunk 1]
```diff
@@ -1,16 +1,191 @@
...
+
+```html
+<link rel="manifest" href="/manifest.json">
+```
+
+Create your manifest with any text editor or IDE - just make sure your JSON is valid. Start with a minimal manifest to get a working PWA. Then iteratively add new [members](https://aka.ms/learn-PWA/30Days-1.3/developer.mozilla.org/docs/Web/Manifest) to improve PWA experience. Then add capability-driven attributes related to [Web APIs](https://aka.ms/learn-PWA/30Days-1.3/developer.mozilla.org/en-US/docs/Web/API) you support.
+
+Audit your PWA with tools like [PWABuilder](https://aka.ms/learn-PWA/30Days-1.3/pwabuilder.com) to find, and fix, issues in your manifest so you can deliver a best-in-class PWA experience to users.
+
+## What does a minimal Manifest need?
```
Comment (zateutsch) [inline docs/30DaysOfPWA/core-concepts/03.md:129]: Should "Manifest" be capitalized here?

File: docs/30DaysOfPWA/core-concepts/03.md [hunk 2]
```diff
@@ -1,16 +1,191 @@
...
+ * `display` - how much of browser UI does user see in app? 
+ * `background_color` - placeholder to show (before stylesheet loads)
+ * `theme_color` - default theme color for app
+ * `scope` - what web pages can be viewed (navigation scope)
+ * `description` - what is the app about?
+ * `icons` - array of icon images used in various contexts
+ * `screenshots` - showcase application UI/UX (e.g., for app stores
+
+
+## What is a "good" Manifest?
```
Comment (zateutsch) [inline docs/30DaysOfPWA/core-concepts/03.md:153]: Here as well.

File: docs/30DaysOfPWA/core-concepts/03.md [hunk 3]
```diff
@@ -1,16 +1,191 @@
...
+
+One last thought. Remember our analogy for the manifest being a resume for the PWA? Where manifest "members" reflect different sections related to identity, characteristics and capability? 
+
+Think of _web capabilities_ and other open web technologies (e.g, Service Workers) as being a **rolodex** of team-mates that this app can call upon, to get the work done. They can now list those skills on their resume because it's a skill they are _capable_ of. However, their ability to do that job on a given device is dependent on the _availability_ of those teammates. Think of "feature detection" as equivalent to _making a call_ to see if the required colleague is around before committing to support that task.
+
+---
+
+## Exercise: Inspect Sample PWA Manifest
+
+We covered a lot today - and now it's your turn to learn by actively exploring the concepts. Pick a different Sample App.
```
Comment (zateutsch) [inline docs/30DaysOfPWA/core-concepts/03.md:181]:
```suggestion
We covered a lot today - and now it's your turn to learn by actively exploring the concepts. Pick a different [Sample App.]((https://aka.ms/learn-PWA/30Days-1.3/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/demo-pwas)
```

Link to sample apps again in Exercise.

File: docs/30DaysOfPWA/core-concepts/03.md [hunk 4]
```diff
@@ -1,16 +1,191 @@
...
+
+One last thought. Remember our analogy for the manifest being a resume for the PWA? Where manifest "members" reflect different sections related to identity, characteristics and capability? 
+
+Think of _web capabilities_ and other open web technologies (e.g, Service Workers) as being a **rolodex** of team-mates that this app can call upon, to get the work done. They can now list those skills on their resume because it's a skill they are _capable_ of. However, their ability to do that job on a given device is dependent on the _availability_ of those teammates. Think of "feature detection" as equivalent to _making a call_ to see if the required colleague is around before committing to support that task.
```
Comment (zateutsch) [inline docs/30DaysOfPWA/core-concepts/03.md:175]: Iffy on the metaphor when it gets to the rolodex part. I had to google rolodex, might miss the mark if our intended audience is largely students.

**Review by zateutsch — APPROVED** [review]
Love it!

A few minor things.

File: docs/30DaysOfPWA/core-concepts/03.md [hunk 5]
```diff
@@ -1,16 +1,191 @@
...
+
+Plus, we picked a [Sample PWA](https://aka.ms/learn-PWA/30Days-1.3/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/demo-pwas) to look at as we explore the concepts. Now, let's dive into PWA building blocks, starting with manifests.
+
+## What is a Web App Manifest?
+
+_Installability_ is a desirable PWA characterstic that allows it to be added to a device home screen and behave like other native apps - e.g., users can launch them, pin them to Taskbar, find them through device search etc.
+
+The **Web App Manifest** is an open web [specification of a JSON format](https://aka.ms/learn-PWA/30Days-1.3/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/web-app-manifests) that is critical to making PWAs installable. [Functionally](https://aka.ms/learn-PWA/30Days-1.3/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/web-app-manifests), it _governs how your PWA looks and behaves when installed on device_ by defining the properties (key-value pairs) that characterize its appearance and behaviors.
```
Comment (JudahGabriel) [inline docs/30DaysOfPWA/core-concepts/03.md:31]: > The Web App Manifest is an open web specification of a JSON format that is critical to making PWAs installable, it governs how your PWA looks and beahves when installedon device by defining the properties (key-value pairs) that characterize its appearance and behaviors.

This reads a bit mechanical. I'd recommend a slight change to that to what is to the application. Something like,

> The Web App Manifest is a JSON file that describes details about your web app, such as your app name, description, icons, and more.

File: docs/30DaysOfPWA/core-concepts/03.md [hunk 6]
```diff
@@ -1,16 +1,191 @@
...
+
+
+## A Sample PWA Manifest
+
+Let's take a look at what a real manifest looks like using a [Sample PWA](https://aka.ms/learn-PWA/30Days-1.3/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/demo-pwas). I'm using [DevTool Tips](https://aka.ms/learn-PWA/30Days-1.3/devtoolstips.org) - and I can view its manifest in two ways:
+ * **From runtime.** In [yesterday's post](02.md) we learned to _inspect_ the app with browser DevTools to find the Manifest section under the Application tab.
+ * **From source.** Look for a [manifest.json](https://aka.ms/learn-PWA/30Days-1.3/raw.githubusercontent.com/captainbrosset/devtools-tips/main/src/manifest.json) file. Apps may choose to use a `app_name.webmanifest` naming convention if they serve it with an `application/manifest+json` MIME type.
```
Comment (JudahGabriel) [inline docs/30DaysOfPWA/core-concepts/03.md:38]: In the "From source" section, you may want to mention you can find a `<link rel="manifest" href="manifest.json">` inside the `<head>` of the page. The `href` will specify the file name of the manifest file.

File: docs/30DaysOfPWA/core-concepts/03.md [hunk 7]
```diff
@@ -1,16 +1,191 @@
...
+
+Create your manifest with any text editor or IDE - just make sure your JSON is valid. Start with a minimal manifest to get a working PWA. Then iteratively add new [members](https://aka.ms/learn-PWA/30Days-1.3/developer.mozilla.org/docs/Web/Manifest) to improve PWA experience. Then add capability-driven attributes related to [Web APIs](https://aka.ms/learn-PWA/30Days-1.3/developer.mozilla.org/en-US/docs/Web/API) you support.
+
+Audit your PWA with tools like [PWABuilder](https://aka.ms/learn-PWA/30Days-1.3/pwabuilder.com) to find, and fix, issues in your manifest so you can deliver a best-in-class PWA experience to users.
+
+## What does a minimal Manifest need?
+
+The [specfication](https://aka.ms/learn-PWA/30Days-1.3/w3c.github.io/manifest#examples) defines a number of _member_ attributes - but a suggested [minimal manifest](https://aka.ms/learn-PWA/30Days-1.3/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/web-app-manifests) should have at least these three - where *start_url* defines the entry point (default path shown) when app is launched on device.
+
+```json
```
Comment (JudahGabriel) [inline docs/30DaysOfPWA/core-concepts/03.md:133]: I'd also recommend at least 1 icon as a minimum. Google Play and Microsoft Store require it.

File: docs/30DaysOfPWA/core-concepts/03.md [hunk 8]
```diff
@@ -1,16 +1,191 @@
...
+{
+    "name": "My Sample PWA",
+    "lang": "en-US",
+    "start_url": "/"
+}
+```
+
+A user-friendly description of the supported members can be read [here](https://aka.ms/learn-PWA/30Days-1.3/developer.mozilla.org/docs/Web/Manifest) - let's see what our sample PWA is declaring for instance:
+ * `short_name` - app name for constrained spaces (e.g., home screen)
```
Comment (JudahGabriel) [inline docs/30DaysOfPWA/core-concepts/03.md:142]: Maybe mention `short_name` should be 12 characters or less, per Google's PWA guidelines. It's used in places like the Android home screen.

File: docs/30DaysOfPWA/core-concepts/03.md [hunk 9]
```diff
@@ -1,16 +1,191 @@
...
+ * `categories` - hints for stores or app catalogs. (See W3C [examples](https://aka.ms/learn-PWA/30Days-1.3/github.com/w3c/manifest/wiki/Categories))
+ * `display` - how much of browser UI does user see in app? 
+ * `background_color` - placeholder to show (before stylesheet loads)
+ * `theme_color` - default theme color for app
+ * `scope` - what web pages can be viewed (navigation scope)
+ * `description` - what is the app about?
+ * `icons` - array of icon images used in various contexts
+ * `screenshots` - showcase application UI/UX (e.g., for app stores
```
Comment (JudahGabriel) [inline docs/30DaysOfPWA/core-concepts/03.md:150]: Missing end parens.

**Review by JudahGabriel — CHANGES_REQUESTED** [review]

File: docs/30DaysOfPWA/core-concepts/03.md (hunk 2 above)
Comment (JudahGabriel) [reply docs/30DaysOfPWA/core-concepts/03.md:153]: I wouldn't use quotes for good. Italics maybe, or even unstyled is fine.

File: docs/30DaysOfPWA/core-concepts/03.md (hunk 5 above)
Comment (beth-panx) [inline docs/30DaysOfPWA/core-concepts/03.md:31]:
```suggestion
The **Web App Manifest** is an open web [specification of a JSON format](https://aka.ms/learn-PWA/30Days-1.3/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/web-app-manifests) that is critical to making PWAs installable. [Functionally](https://aka.ms/learn-PWA/30Days-1.3/docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/web-app-manifests), it _governs how your PWA looks and behaves when installed on device_ by defining the properties (key-value pairs) that characterize its appearance and behaviors. In other words, it describes details about your web app, such as your app name, description, icons, and more. 
```

File: docs/30DaysOfPWA/core-concepts/03.md (hunk 6 above)
Comment (beth-panx) [inline docs/30DaysOfPWA/core-concepts/03.md:38]:
```suggestion
 * **From source.** Look for a [manifest.json](https://aka.ms/learn-PWA/30Days-1.3/raw.githubusercontent.com/captainbrosset/devtools-tips/main/src/manifest.json) file. Apps may choose to use a `app_name.webmanifest` naming convention if they serve it with an `application/manifest+json` MIME type. You can find a `<link rel="manifest" href="manifest.json">` inside the `<head>` tag of the page. The `href` will specify the file name of the manifest file.
```

File: docs/30DaysOfPWA/core-concepts/03.md (hunk 1 above)
Comment (beth-panx) [inline docs/30DaysOfPWA/core-concepts/03.md:129]:
```suggestion
## What does a minimal manifest need?
```

File: docs/30DaysOfPWA/core-concepts/03.md (hunk 9 above)
Comment (beth-panx) [inline docs/30DaysOfPWA/core-concepts/03.md:150]:
```suggestion
 * `screenshots` - showcase application UI/UX (e.g., for app stores)
```

File: docs/30DaysOfPWA/core-concepts/03.md (hunk 2 above)
Comment (beth-panx) [inline docs/30DaysOfPWA/core-concepts/03.md:153]:
```suggestion
## What is a _good_ manifest?
```

File: docs/30DaysOfPWA/core-concepts/03.md (hunk 8 above)
Comment (beth-panx) [reply docs/30DaysOfPWA/core-concepts/03.md:142]: @JudahGabriel I couldn't find the reference of '12 char or less'. Mind sharing a link that we can point to?

File: docs/30DaysOfPWA/core-concepts/03.md (hunk 7 above)
Comment (beth-panx) [reply docs/30DaysOfPWA/core-concepts/03.md:133]: I think this point can be brought up later in the series when the PWA is intended for the store.

File: docs/30DaysOfPWA/core-concepts/03.md (hunk 4 above)
Comment (beth-panx) [reply docs/30DaysOfPWA/core-concepts/03.md:175]: I too googled this! cc/ @nitya for visibility. Won't change it here in the PR.

**Pull request merged**